src/scFetch.js

```diff
@@ -0,0 +1,46 @@
+import SC from 'soundcloud';
+import _ from 'lodash';
+
+const DEFAULT_USER_ID = '7742327';
+
+SC.initialize({
+  client_id: process.env.REACT_APP_SC_CLIENT_ID,
+  redirect_uri: process.env.REACT_APP_REDIRECT_URI,
+});
+
+export const getTracks = () => {
+  if (process.env.REACT_APP_ENV === 'development') {
+    return SC.get(`/users/${DEFAULT_USER_ID}/favorites`)
+      .then(favorites => favorites)
+      .catch(err => { throw new Error(`scFetch-development getTracks: ${err.message}`)});
```

Comment:
Non-blocker: If you're not going to do something with it, don't catch it. All you're doing here is throwing away all the interesting fields on `err` for a custom message.